projects/golang/go/build/conformanceTests/eksDistroRebuildBuildspec.yaml

```diff
@@ -0,0 +1,11 @@
+version: 0.2
```

Comment:
this file is just a stub; it'll evolve as we work on the eks-distro rebuild system.